src/components/VirtualizedList/index.tsx

```diff
@@ -29,11 +29,16 @@ const VirtualizedList: FunctionComponent<VirtualizedListProps> = ({
   const Row: FunctionComponent<ListChildComponentProps> = ({ index, style }) => (
     <>
       {addSpacerRow && index === 0 && <div style={style} />}
-      {(!addSpacerRow || (addSpacerRow && index !== 0)) && (
+      {(!addSpacerRow || (addSpacerRow && index !== 0)) && localRecords[index].id && (
         <Link to={`/details/${localRecords[index].id}`} className={`ListRow ${s.moditeRow}`} style={style}>
           <ModiteListItem item={localRecords[index]} key={localRecords[index].id} />
         </Link>
       )}
+      {(!addSpacerRow || (addSpacerRow && index !== 0)) && !localRecords[index].id && (
+        <span className={`ListRow ${s.moditeRow}`} style={style}>
+          <ModiteListItem item={localRecords[index]} />
+        </span>
+      )}
```

Comment:
When does not have an id (e.g. no modite found) there is no detail link.

Reply:
can you add that as a comment in the code?

Reply:
Sure. Just dit!

Was very nice playing around React for the first time. Hope I will be able to contribute more.